pkgs/applications/radio/yaac/default.nix

```
@@ -0,0 +1,34 @@
+{ makeWrapper, lib, stdenv, fetchurl, unzip, jre8 }:
+
+stdenv.mkDerivation rec {
+  version = "1.0-beta181";
+  pname = "YAAC";
```

```suggestion
  pname = "yaac";
```
> The pname and the version attribute must not contain uppercase letters

https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

---

Updated in latest.